packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.js

```diff
@@ -0,0 +1,83 @@
+import { Box, DropButton } from 'grommet'
+import { Filter } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React, { useRef, useState } from 'react'
+
+import Characteristics from '../Characteristics'
+import FilterButton from '../Characteristics/components/FilterButton'
+
+export default function FilterStatus (props) {
+  const { task } = props
+  const filterStatusRef = useRef()
+
+  // TODO: refactor filter state to model
+  const [ filters, setFilters ] = useState({})
```

Comment:
Filter state could be a React component state on the in a container component for Chooser?

Reply:
With filters selected saved with the annotation, I was thinking I'd revisit when wiring components together with store?

Reply:
I'm still not sure that's necessary, but we can chat about that at our call tomorrow?

Reply:
Per call plan is to keep filter state as React state and move state to SurveyTask (soon to be) container.